src/loaders/mongoose.js

```diff
@@ -1,17 +1,31 @@
 const mongoose = require("mongoose");
+const { GridFSBucket } = require("mongodb");
 
 const CONFIG = require("../constants/config");
+const CONSTANT = require("../constants/constants");
 
-async function mongooseLoader() {
-  try {
-    await mongoose.connect(CONFIG.MONGODB_ENDPOINT, {
-      DBName: "Figci",
-    });
+let bucket;
 
-    console.log("connected to database");
+const connectMongoDB = async () => {
+  try {
+    await mongoose.connect(CONFIG.MONGODB_ENDPOINT, { dbName: CONFIG.DB_NAME });
   } catch (error) {
-    console.error(error);
+    console.error("MongoDB connection error:", error);
+
+    throw error;
   }
-}
+};
+
+const initBucket = async () => {
+  const db = mongoose.connection.client.db(CONFIG.DB_NAME);
+  bucket = new GridFSBucket(db, { bucketName: CONSTANT.BUCKET_NAME });
```

Comment:
- `P1`
선언부와 비즈니스 로직은 한줄 개행하는건 어떨까요!

```suggestion
  const db = mongoose.connection.client.db(CONFIG.DB_NAME);
  
  bucket = new GridFSBucket(db, { bucketName: CONSTANT.BUCKET_NAME });
```